code/datums/ruins/icemoon.dm

```diff
@@ -13,7 +13,7 @@
 	description = "Blown up by an unfortunate accident."
 	suffix = "icemoon_surface_engioutpost.dmm"
 
-/datum/map_template/ruin/icemoon/slimerancher //Shiptest edit
```

Comment:
I feel like it's incredibly unnecessary to wipe shiptest out of comments, that removes a significant amount of context from "why was this decision made?"